sdk/program/src/pubkey.rs

```diff
@@ -668,6 +668,48 @@ impl fmt::Display for Pubkey {
     }
 }
 
+impl borsh0_9::de::BorshDeserialize for Pubkey {
```

Comment:
Maybe add a TODO to use `borsh` via a proxy crate?

Reply:
Great idea, put in #32668 to address the work